Squest/ldap_config.py

```diff
@@ -6,18 +6,20 @@
 # -----------------------
```

Comment:
This file is an example. Each user ned to configure it following his env. 
We don't use "env" variable as it would be hard to cover all flag proposed by the plugin.

Reply:
It could be overided by users using volumes, but it should be included in repo with most of options covered by envs.

Reply:
So variables should be in the doc.
And both usage need to be documented.

Reply:
Done